_episodes/13-scheduler.md

```diff
@@ -4,11 +4,13 @@ teaching: 45
 exercises: 30
 questions:
 - "What is a scheduler and why are they used?"
-- "How do we submit a job?"
+- "How do I launch a program to run on any one remote host in the cluster?"
+- "How do I capture the output of a program that is run on a remote host?"
```

Comment:
Maybe use "computer" instead of "remote host"?

Reply:
I'd personally would favor `node`, see 2b7de7a. what do you think?

Reply:
That sounds good to me too.